### ISSUE TITLE
Account Update not working

### ISSUE THREAD

**Issue by nil1511** [adcpm/steem#50]
AccountUpdate is not working on master. 
check here: https://dev.steemconnect.com/profile

```
"Error: transaction.operations = accountUpdate,[object Object]	 cause: Illegal value:undefined (not an integer) 
 stack: TypeError: Illegal value: undefined (not an integer)
    at i.a.writeVarint32 (https://dev.steemconnect.com/js/app.min.js:65:17946)
    at Object.appendByteBuffer (https://dev.steemconnect.com/js/app.min.js:113:26194)
    at Object.appendByteBuffer (https://dev.steemconnect.com/js/app.min.js:113:20805)
    at e.appendByteBuffer (https://dev.steemconnect.com/js/app.min.js:73:10259)
    at e.toByteBuffer (https://dev.steemconnect.com/js/app.min.js:73:11937)
    at e.toBuffer (https://dev.steemconnect.com/js/app.min.js:73:12030)
    at Object.p.signTransaction (https://dev.steemconnect.com/js/app.min.js:114:972)
    at https://dev.steemconnect.com/js/app.min.js:113:2736
    at i (https://dev.steemconnect.com/js/app.min.js:59:8507)
    at i._settlePromiseFromHandler (https://dev.steemconnect.com/js/app.min.js:58:13798)
    at i._settlePromise (https://dev.steemconnect.com/js/app.min.js:58:14601)
    at i._settlePromise0 (https://dev.steemconnect.com/js/app.min.js:58:15302)
    at i._settlePromises (https://dev.steemconnect.com/js/app.min.js:58:16632)
    at i._fulfill (https://dev.steemconnect.com/js/app.min.js:58:15673)
    at i._resolveCallback (https://dev.steemconnect.com/js/app.min.js:58:12545)
    at i._settlePromiseFromHandler (https://dev.steemconnect.com/js/app.min.js:58:13953) 
    at i.a.writeVarint32 (https://dev.steemconnect.com/js/app.min.js:65:17946)
    at Object.appendByteBuffer (https://dev.steemconnect.com/js/app.min.js:113:26194)
    at Object.appendByteBuffer (https://dev.steemconnect.com/js/app.min.js:113:20805)
    at e.appendByteBuffer (https://dev.steemconnect.com/js/app.min.js:73:10259)
    at e.toByteBuffer (https://dev.steemconnect.com/js/app.min.js:73:11937)
    at e.toBuffer (https://dev.steemconnect.com/js/app.min.js:73:12030)
    at Object.p.signTransaction (https://dev.steemconnect.com/js/app.min.js:114:972)
    at https://dev.steemconnect.com/js/app.min.js:113:2736
    at a._throw (https://dev.steemconnect.com/js/app.min.js:73:12537)
    at e.appendByteBuffer (https://dev.steemconnect.com/js/app.min.js:73:10374)
    at e.toByteBuffer (https://dev.steemconnect.com/js/app.min.js:73:11937)
    at e.toBuffer (https://dev.steemconnect.com/js/app.min.js:73:12030)
    at Object.p.signTransaction (https://dev.steemconnect.com/js/app.min.js:114:972)
    at https://dev.steemconnect.com/js/app.min.js:113:2736
From previous event:
    at Object.w.send (https://dev.steemconnect.com/js/app.min.js:113:2607)
    at Object.w.(anonymous function) [as accountUpdateWith] (https://dev.steemconnect.com/js/app.min.js:113:3638)
    at Object.w.(anonymous function) [as accountUpdate] (https://dev.steemconnect.com/js/app.min.js:113:4097)
    at https://dev.steemconnect.com/js/app.min.js:16:28633
    at https://dev.steemconnect.com/js/app.min.js:16:28595
    at https://dev.steemconnect.com/js/app.min.js:112:29801
    at Object.accountUpdate (https://dev.steemconnect.com/js/app.min.js:112:30562)
    at t.n.updateProfile (https://dev.steemconnect.com/js/app.min.js:75:11115)
    at Object.onEnter (https://dev.steemconnect.com/js/app.min.js:75:11718)
    at r.onEnter (https://dev.steemconnect.com/js/app.min.js:32:21371)
    at Object.r (https://dev.steemconnect.com/js/app.min.js:18:27726)
    at a (https://dev.steemconnect.com/js/app.min.js:18:23809)
    at Object.s [as executeDispatchesInOrder] (https://dev.steemconnect.com/js/app.min.js:18:24024)
    at h (https://dev.steemconnect.com/js/app.min.js:17:3925)
    at v (https://dev.steemconnect.com/js/app.min.js:17:4051)
    at Array.forEach (native)
    at n (https://dev.steemconnect.com/js/app.min.js:72:20296)
    at Object.processEventQueue (https://dev.steemconnect.com/js/app.min.js:17:5129)
    at r (https://dev.steemconnect.com/js/app.min.js:110:21998)" 
```

**Comment by yamadapc** [comment]
Fixed by 6ce1a43afa34a5c82509c3f15cf3cbb8642c0f3a

**Comment by yamadapc** [comment]
This is related to adcpm/steemauth#7
